config/viper_source_test.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/require"
+)
+
+func TestViperSource(t *testing.T) {
+	viper.SetConfigType("TOML")
+	cosiRate := "10s"
+
+	var tomlExample = []byte(`
+[server]
+ip = "0.0.0.0"
+
+[cosi]
+rate = "10s"
```

Comment:
We need to find a better example - the rate is given in `s**-1`, not `s`: is this 10 signatures per second or one signature every 10 seconds?